proposals/0022-multi-stake.md

```diff
@@ -0,0 +1,183 @@
+---
+simd: '0022'
+title: Multi Delegation Stake Account
+authors:
+  - Jon Cinque (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-01-20
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+A new "multi stake" account that allows for multiple movements of stake, useful
+for simpler stake movement through more activations, deactivations, or redelegation.
+
+## New Terminology
+
+* "multi stake": a stake account with multiple `Delegation` instances
+* "small stake": a stake whose delegation is a small lamport amount
+
+## Motivation
+
+Stake operations are cumbersome or impossible for many ordinary uses. For example,
+delegating more lamports in an existing stake account requires creating a new
+account, delegating, waiting, merging, then withdrawing the rent-exempt lamports.
+
+The current stake `redelegate` instruction requires using a new stake account and
+eventually cleaning up the old one, which can be tricky to use.
+
+As a minimum stake delegation amount is applied to the network, and potentially
+increased over time, these operations will become more complicated, since all
+delegations must clear that threshold. A lot of small stake will be left liquid.
```

Comment:
```suggestion
delegations must clear that threshold. A lot of small stake will be left illiquid.
```

Reply:
Or is this referring to not being able to withdraw stake from pools due min. stake enforcement?